tests/Admin/AdminTest.php

```diff
@@ -1440,14 +1441,17 @@ public function testTransChoiceWithMessageDomain(): void
 
     public function testSetFilterPersister(): void
     {
-        $admin = new PostAdmin('sonata.post.admin.post', 'NewsBundle\Entity\Post', 'SonataNewsBundle:PostAdmin');
+        $admin = new class('sonata.post.admin.post', 'NewsBundle\Entity\Post', 'SonataNewsBundle\PostAdminController') extends PostAdmin {
```

Comment:
```suggestion
        $admin = new class('sonata.post.admin.post', 'NewsBundle\Entity\Post', 'SonataNewsBundle\Controller\PostAdminController') extends PostAdmin {
```

Sorry, this should be the location for the corresponding admin controller